src/App.js

```diff
@@ -1,25 +1,19 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Router, Route } from 'react-router-dom'
+import { Grommet } from 'grommet'
+import history from './history'
+import theme from 'theme'
+import Home from './screens/Home'
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router history={history}>
+        <main>
+          <Grommet theme={theme}>
```

Comment:
```suggestion
          <Grommet full theme={theme}>
```

This still needs the suggestion I gave last comment to ensure full viewport width and height.